Widgets/Site.Master

```diff
@@ -0,0 +1,39 @@
+﻿<%@ Master Language="C#" AutoEventWireup="true" CodeBehind="Site.master.cs" Inherits="Widgets.Site" %>
+
+<!DOCTYPE html>
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head runat="server">
+    <title></title>
+    <asp:ContentPlaceHolder ID="head" runat="server">
+    </asp:ContentPlaceHolder>
+</head>
+<body>
+    <form id="form1" runat="server">
+        <header>
+            <asp:Menu ID="MainMenu" runat="server" Orientation="Horizontal">
+                <Items>
+                    <asp:MenuItem NavigateUrl="~/Default.aspx" Text="Home" Value="Home" />
+                    <asp:MenuItem NavigateUrl="~/CounterExample.aspx" Text="Counter" Value="Counter" />
+                    <asp:MenuItem NavigateUrl="~/FontChangeExample.aspx" Text="Font Preview" Value="Font Preview" />
+
+                    <asp:MenuItem NavigateUrl="~/QuoteOfTheDayExample.aspx" Text="Quote of the Day" Value="Quote of the Day" />
+                    <asp:MenuItem NavigateUrl="~/ExampleRenderTime.aspx" Text="Render Time" Value="Render Time" />
+                    <asp:MenuItem NavigateUrl="~/WeatherExample.aspx" Text="Weather" Value="Weather" />
+                </Items>
+            </asp:Menu>
+            <span style="margin-right: 5px;">Theme</span>
+            <asp:DropDownList OnSelectedIndexChanged="ThemeDropDownList_SelectedIndexChanged" runat="server" ViewStateMode="Enabled" ID="ThemeDropDownList" AutoPostBack="true" Style="float: right;">
```

Comment:
The `ViewStateMode` property should default to `Enabled`, so you can safely remove this property from this element.